scarb/tests/proc_macro_server.rs

```diff
@@ -36,3 +39,37 @@ fn defined_macros() {
     assert_eq!(response.inline_macros, vec!["inline_some".to_string()]);
     assert_eq!(response.executables, vec!["some_executable".to_string()]);
 }
+
+#[test]
+fn expand_attribute() {
+    let t = TempDir::new().unwrap();
+    let plugin_package = t.child("some");
+
+    CairoPluginProjectBuilder::default()
+        .lib_rs(SIMPLE_MACROS)
+        .build(&plugin_package);
+
+    let project = t.child("test_package");
+
+    ProjectBuilder::start()
+        .name("test_package")
+        .version("1.0.0")
+        .lib_cairo("")
+        .dep("some", plugin_package)
+        .build(&project);
+
+    let mut proc_macro_server = ProcMacroClient::new(&project);
+
+    let item = TokenStream::new("fn some_test_fn(){}".to_string());
+
+    let response = proc_macro_server
+        .request_and_wait::<ExpandAttribute>(ExpandAttributeParams {
+            attr: "some".to_string(),
+            args: TokenStream::empty(),
+            item: item.clone(),
+        })
+        .ok();
+
+    assert_eq!(response.diagnostics, vec![]);
+    assert_eq!(response.token_stream, item);
```

Comment:
Can we do anything in the macro so this test actually tests if the logic executes?

Reply:
I think asserting on the debug-print will be enough

Reply:
>I think asserting on the debug-print will be enough

Will not as these are not proc-macros